src/main/java/uk/gov/di/ipv/cri/common/library/util/SignedJWTFactory.java

```diff
@@ -58,11 +58,13 @@ private JWSHeader generateHeader() {
     private JWSHeader generateHeader(String issuer, String signingKeyId)
             throws NoSuchAlgorithmException {
 
+        issuer = issuer.replaceFirst("https://", "");
```

Comment:
I suppose this works as well, would have done

new URL(issuer).getHost()

Reply:
Hmm what do you reckon, this is a 1 liner. `new URL(issuer).getHost()` would break if the issuer already was just the host and involve exception handling?

Reply:
Just something to think about, maybe your way is better. I think the issuer never truly the host, not sure would require looking to specs so I think stick you method